plugin-k8s/src/lib.rs

```diff
@@ -84,69 +104,153 @@ impl AlumetPlugin for K8sPlugin {
         let plugin_name = self.name().to_owned();
 
         // let metrics = self.metrics.clone().unwrap();
-        let metrics = self.metrics.clone().unwrap();
+        let metrics = self.metrics.clone().with_context(|| "Metrics is not available")?;
```

Comment:
Shouldn't we use `expect` here? The plugin is broken if `self.metrics` is empty at this point.